app/components/layouts/BackNavigation.tsx

```diff
@@ -6,22 +6,23 @@ import { useHistory } from "react-router-dom";
 // Renders a smart back link that handles pages visited directly or from referring page
 export const BackNavigation = ({
   defaultReturnTo,
+  content,
 }: {
-  defaultReturnTo: string;
+  defaultReturnTo?: string;
```

Comment:
Per [suggestion to make this required](https://github.com/Exygy/askdarcel-web/pull/66/files#r1661475921), can we use PropTypes in functional components? It's been a while since I did this. 

```suggestion
  defaultReturnTo: PropTypes.string.isRequired.
```